views/eApostilles/applicationSubmissionSuccessful.ejs

```diff
@@ -1 +1,41 @@
-<h1 class="heading-xlarge">Application successfully submitted</h1>
\ No newline at end of file
+<% pageTitle="e-Apostille application submitted" %>
+
+<div class="inner_header no-user-signed-in">
+    <%- partial ('../partials/inner-header.ejs')%>
+</div>
+
+<section class="column-two-thirds">
+    <div class="container-confirmed govuk-!-margin-bottom-5">
+        <h1 class="heading-xlarge confirmed">Application submitted</h1>
+        <div class="heading-large govuk-!-margin-top-0 govuk-!-margin-bottom-0" style="font-weight: normal">
```

Comment:
can we provide the inline style via css?

Reply:
sure thing, I'll try and BEMify it as well 😄 
Then again, for a one-liner utility class like this is it worth using BEM for this 🤔

Reply:
you can use one of govuk's utility classes - `govuk-!-font-weight-regular`

Reply:
Ah didn't know this existed. I'll do that instead, thanks 👍